src/compas/data/data.py

```diff
@@ -28,15 +28,77 @@
 class Data(object):
     """Abstract base class for all COMPAS data objects.
 
+    Parameters
+    ----------
+    name : str, optional
+        The name of the object.
+
     Attributes
     ----------
-    DATASCHEMA : :class:`schema.Schema`
-        The schema of the data dict.
-    JSONSCHEMA : dict
-        The schema of the serialized data dict.
+    dtype : str, read-only
+        The type of the object in the form of a fully qualified module name and a class name, separated by a forward slash ("/").
+        For example: ``"compas.datastructures/Mesh"``.
     data : dict
-        The fundamental data describing the object.
-        The structure of the data dict is defined by the implementing classes.
+        The representation of the object as a dictionary containing only native Python data.
```

Comment:
I would make this one very clear about what the dictionary should contain, since I remember being confused myself multiple times. Does `only native Python data` mean `only Python built-in data types`? Because an instance of a user-created class would still qualify as native Python in some definitions, so, if we want to avoid that, I would go for the "built-in data type" wording.